sys/include/net/netstats.h

```diff
@@ -18,13 +18,22 @@
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  */
 
+#include <stdint.h>
+
 #ifndef NETSTATS_H
 #define NETSTATS_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef enum {
+    NETSTATS_LAYER2     = 0x01,
+    NETSTATS_IPV6       = 0x02,
+    NETSTATS_ALL        = 0xFF
```

Comment:
Not sure about the compiler optimization, but what about choosing `0` for `NETSTATS_ALL` to have contiguous numbers for the `switch case` below? This might increase the chances of using a jump table via function pointers by the copmiler, instead of a `if-else-if-` chain..